operator/webapp/core/test/test_k8s_client.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from kubernetes import client
+from core.k8s_client import create_integration_route, create_route_configmap
+from models import RouteData, Resource, Status
+from kubernetes.client.rest import ApiException
+
+
+@pytest.fixture
+def route_data():
+    return RouteData(
+        namespace="default",
+        route_name="my-route",
+        route_file="<xml>payload</xml>",
+    )
+
+
+@pytest.fixture(autouse=True)
```

Comment:
:question: Is the `autouse=True` required? Looks the following tests are already requesting the fixture explicitly.

Reply:
No, we don't need this